src/common/test-webserver.c

```diff
@@ -81,7 +81,7 @@ setup (TestCase *tc,
 
   /* Automatically chosen by the web server */
   g_object_get (tc->web_server, "port", &port, NULL);
-  tc->localport = g_strdup_printf ("localhost:%d", port);
+  tc->localport = g_strdup_printf ("127.0.0.1:%d", port);
```

Comment:
We're going to take this as a workaround for now, but want to revert it later after some glib changes make it into the distros.